G-licious/Controllers/StationsViewController.m

```diff
@@ -1,405 +0,0 @@
-//
```

Comment:
Can you make an issue with that commit number and set a milestone for the 4.0.0 (or whatever the target version for the successor of this portal-app) release?